webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -4,11 +4,28 @@
 from django.http import HttpResponseBadRequest, Http404
 from django.shortcuts import redirect
 from django.core.urlresolvers import reverse
+from django.shortcuts import render
 
 from . import utils
 from crashstats.base import ga
 
 
+def decorate_bad_request(view):
```

Comment:
A decorator named ``decorate`` is not very helpful to me. What about ``handle`` or ``catch`` instead?